tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -163,10 +174,81 @@
                        subjects_dir=subjects_dir, dig=True,
                        surfaces=['head-dense', 'white'], coord_frame='meg')
 
+
 ###############################################################################
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+#
+# First, let's define a helper function to plot the alignment of the
+# points with the head using voxel coordinates.
+
+def plot_dig_alignment(points, coord_system='ras'):
+    renderer_kwargs = dict(bgcolor='w', smooth_shading=False)
+    renderer = mne.viz.backends.renderer._get_renderer(
+        size=(800, 400), **renderer_kwargs)
+    seghead_rr, seghead_tri = mne.read_surface(os.path.join(
+        subjects_dir, 'sample', 'surf', 'lh.seghead'))
+    renderer.mesh(*seghead_rr.T, triangles=seghead_tri, color=(0.7,) * 3,
+                  opacity=0.2)
+    for point in points.copy():
+        if coord_system == 'vox':
+            point -= 128  # RAS zero centered
+            center = (point[0], point[2], -point[1])
+        else:
+            center = point
+        renderer.sphere(center=center, color='r', scale=5)
+    view_kwargs = dict(elevation=90, azimuth=0)
+    view_kwargs['focalpoint'] = (0., 0., 0.)
+    mne.viz.set_3d_view(figure=renderer.figure, distance=1000, **view_kwargs)
+    renderer.show()
+
+
+###############################################################################
+# Get landmarks in head space from the DigMontage stored in raw
+# -------------------------------------------------------------
+# Adjust units by * 1e3 m -> mm.
+#
+# The plot below shows the unaligned digitization points in the
+# coordinate space native to the digitizer equipment.
```

Comment:
```suggestion
# coordinate space native to the digitizer equipment plotted with
# the head surface in the MRI voxel coordinate frame. These coordinate
# frames are are not the same, so there is a mismatch.
```
or something like this. I think it's important to make clear what frame you're plotting each in from the start, then as you transform below you get them to the same coordinate frame and they match